src/entity/Bullet.java

```diff
@@ -1,74 +1,69 @@
 package entity;
 
 import java.awt.Color;
-
 import engine.DrawManager.SpriteType;
 
 /**
  * Implements a bullet that moves vertically up or down.
  * 
- * @author <a href="mailto:RobertoIA1987@gmail.com">Roberto Izquierdo Amo</a>
- * 
+ * @author ...
  */
 public class Bullet extends Entity {
 
-	/**
-	 * Speed of the bullet, positive or negative depending on direction -
-	 * positive is down.
-	 */
-	private int speed;
```

Comment:
The speed of the bullet should be kept as an integer because the x and y positions are integers